### DISCUSSION TITLE
How can I delete an entry?

### DISCUSSION THREAD

**Discussion by TheWilley** [IgnisDa/ryot#321] [Q&A] [answered]
Hello! I want to begin by saying that I've enjoyed playing around with the app, and it's been an absolutely amazing experience so far. Now, I may just be missing something obvious here, but is there any way to delete an entry within the app itself? In the homepage, you can create entries easily by clicking on the "Create a media item" button, but I can't seem to find a delete option on any existing entries.

**Comment by IgnisDa** [comment, marked as answer]
Entries are automatically deleted (by cron jobs every 24 hours) based on [these](https://github.com/IgnisDa/ryot/blob/170f51cd7c30dad4dd867a2c77147f6df52693c1/apps/backend/src/migrator/m20230417_create_user.rs#L11-L17) conditions.

**Reply by IgnisDa** [reply]
By entries I mean only media items, genres, and creators (though that will change after #226).

**Reply by TheWilley** [reply]
Ah I see. So basically, if its not used, it's removed. Cool!

Out of curiosity, will there be plans to add a manual delete button? It may make sense if you added an entry by accident or want one gone sooner than 24 hours.

**Reply by IgnisDa** [reply]
I don't really see the point to it, it will just result in useless clutter. The Actions tab already has so many buttons, I am loath to add more without a good reason.

**Reply by IgnisDa** [reply]
Though a graphql mutation is doable.

**Reply by TheWilley** [reply]
Yea that's true, good point!

**Reply by IgnisDa** [reply]
LMK if you really need it.
